recipes/easylzma/all/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 2.6)
 project(cmake_wrapper)
 
-include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
-conan_basic_setup(NO_OUTPUT_DIRS) #Do not call conan_output_dirs_setup(), this paths are set inside the CMakeLists.
```

Comment:
This library defines the output dirs in a weird way inside the CMake. If you change this line the lib folder ends up in the root of the build folder instead of on the folder easylzma-0.0.7 that creates for that purpose. I think is cleaner to maintain the original file structure, so we have the complete output of the build in a single folder.

Reply:
Oh, I only tested it using MSVC.
[You mean this line?](https://github.com/lloyd/easylzma/blob/1753f61f673abdebc08fcf4a37eedd6da8a67ba1/src/CMakeLists.txt#L29)

If it is required for gcc on Linux,
I would just patch this line out because it makes the recipe so much easier.

Reply:
I just tested this pr on Linux and it works fine.

Reply:
Yes, it works but I think it's not clean. From my perspective it's better to maintain the intended file structure.

Reply:
I think it is there to put all binaries in one directory for redistribution.
He should have used cmake `install`.